sinfonier-web/app/views/Modules/module.html

```diff
@@ -98,4 +98,27 @@ <h1 id="module-h1-title">&{'Modules.details.title'} '${module?.getName()} (vc${m
 
     #{script 'util/code.js'/}
     #{script 'modules.js'/}
+

```

Comment:
you can use a data tag in an html tag to store the module id.
<div data-module-id="${_module.getId()}" >